pkg/daemons/config/types.go

```diff
@@ -104,6 +106,30 @@ type Agent struct {
 	EnableIPv6              bool
 }
 
+// ControlQuorumArgs contains parameters that all control plane nodes in HA must share
+type ControlQuorumArgs struct {
+	ClusterDNS               net.IP
+	ClusterDNSs              []net.IP
+	ClusterDomain            string
+	ClusterIPRange           *net.IPNet
+	ClusterIPRanges          []*net.IPNet
+	Disables                 map[string]bool
+	DisableAPIServer         bool
+	DisableCCM               bool
+	DisableControllerManager bool
+	DisableETCD              bool
+	DisableHelmController    bool
+	DisableKubeProxy         bool
+	DisableNPC               bool
+	DisableScheduler         bool
+	DisableServiceLB         bool
```

Comment:
Some of these disables will intentionally be out of sync across servers when using etcd-only or control-plane-only servers. We shouldn't verify anything that's conditionally disabled for that use case.

Reply:
Thanks! I was not aware of that. Could you point me to the ones that could be intentionally out of sync? Are all the ones starting with `Disable...`?

Reply:
All of the ones that are related to etcd or kubernetes components. So etcd, apiserver, controllermanager, scheduler I believe?